tests/integration/proxy/testproxymain.cpp

```diff
@@ -775,6 +797,25 @@ void TestProxyImpl::proxy_created(const Glib::RefPtr<Gio::AsyncResult> result) {
     m_proxy->TestPropReadWriteChar_set('X', sigc::bind(sigc::mem_fun(this, &TestProxyImpl::on_test_prop_read_write_char), 'X'));
     m_proxy->TestPropReadWriteBoolean_set(true, sigc::bind(sigc::mem_fun(this, &TestProxyImpl::on_test_prop_read_write_boolean), true));
 
+    /* Test change notification, for the properties we just changed. */
+    expect_notification(m_proxy->TestPropReadWriteInt_changed());
```

Comment:
Why is TestPropReadWriteInt_changed() expected twice? (On this line and 12 lines below.) I only see a single TestPropReadWriteInt_set() call.

Reply:
Yes, that was wrong :-)